src/libcollectionstest/str.rs

```diff
@@ -1687,6 +1687,19 @@ fn trim_ws() {
                "");
 }
 
+#[test]
+fn to_lowercase() {
+    assert_eq!("".to_lowercase(), "");
+    // https://github.com/rust-lang/rust/issues/26035
+    assert_eq!("'Σ AÉΣ'Σ'' Σ ǅΣ".to_lowercase(), "'σ aéσ'ς'' σ ǆς");
```

Comment:
Ah specifically there's a boolean `is_word_final` and this only exercises one of the two code paths (e.g. only the `true` case I believe)